explainer.md

```diff
@@ -341,70 +348,77 @@ input.accessibleNode.activeDescendant = optionList.accessibleNode;
 
 This would allow the relationship to be expressed naturally.
 
-#### No reflection
+#### AOM and ARIA
+
+While AOM and ARIA both affect the computed accessible properties of a node,
+and have the same vocabulary,
+they are separate interfaces.
+
+##### No reflection
 
-Even though there's correspondence between Accessible Properties and
-ARIA attributes, they are never reflected. Similarly, Accessible Properties
-do not reflect equivalent implicit properties of their associated DOM elements,
-either.
+ARIA does not reflect to AOM:
+
+```html
+<div "clickBtn" role="button">Click here</div>
+```
 
 ```js
-button = document.createElement("button");
-button.innerHTML = "Next";
-console.log(button.accessibleNode.role);   // null, not "button"
-console.log(button.accessibleNode.label);  // null, not "Next"
-
-button.setAttribute("role", "link");
-button.setAttribute("aria-label", "Next Page");
-console.log(button.accessibleNode.role);   // null, not "link"
-console.log(button.accessibleNode.label);  // null, not "Next Page"
+console.log(clickBtn.accessibleNode.role);   // null, not "button"
 ```
 
-Similarly, setting Accessible Properties on an `AccessibleNode` does not
-affect corresponding ARIA attributes at all.
+And AOM does not reflect to ARIA:
 
-```js
-button.accessibleNode.role = "menuitem";
-button.accessibleNode.label = "Next Page (Ctrl+N)";
-console.log(button.getAttribute("role"));  // still "link"
-console.log(button.getAttribute("aria-label"));  // still "Next Page"
+```html
+clickBtn.accessibleNode.role = "link";
+
+console.log(clickBtn.getAttribute("role"));  // Still "button"
 ```
 
-This is analogous to the `style` attribute, which allows
-reading and setting an element's local style, while
-`getComputedStyle()` allows access to the full computed style.
+##### ARIA takes precedence
 
-Part of the reasoning behind this is that there are some cases in which
-Accessible Properties are more expressive than ARIA attributes - for example,
-referencing elements that don't have IDs. Since there would be no way to
-represent those cases as DOM attribute.
+In the case where and AOM Accessible Property and the corresponding ARIA attribute have different values,
+the ARIA attribute takes precedence.
 
-Another reason is for efficiency and tidiness: setting a DOM attribute is more
-heavyweight than setting a property of an object from JavaScript. Authors who
-are concerned about the performance of their web application or the bloated
-size of their generated HTML will find using the Accessibility Object Model
-to be a more efficient alternative. It also promotes creating custom elements
-that are clean and fully encapsulate their own accessibility rather than
-leaking it.
+For example, in the case in the previous section,
+`clickBtn` would have a computed role of `"button"`
+even though its `accessibleNode` has a `role` of `"link"`.
+
+This means that:
```

Comment:
This final paragraph sums up the overall "No reflection" section, but it appears to just be part of the "ARIA takes precedence" section. Maybe it needs a tiny subheading too?